packages/common/src/utils/doctype-utils.ts

```diff
@@ -143,39 +151,51 @@ export class DoctypeUtils {
      * @param record - Record value
      * @param ipfs - IPFS instance
      */
-    static async convertRecordToDTO(record: any, ipfs: IpfsApi): Promise<any> {
+    static async convertRecordToDTO(record: CeramicRecord, ipfs: IpfsApi): Promise<CeramicRecord> {
```

Comment:
maybe I'm being overly pedantic here, but I don't actually think any of these new types are DTOs as they don't actually handle serialization/deserialization off the network.  Because typescript's type checking is so loose, even with this change it's possible to load an object off the network and put it into a CeramicRecord even if it doesn't actually conform to the proper format for any of the CeramicRecord types.  Ideally we would perform some type checking on every object we read off the network, but we don't do that yet.  I already filed https://github.com/ceramicnetwork/js-ceramic/issues/548 for the pub-sub messages, we should probably have a corresponding ticket for the CeramicRecords we load off of IPFS.